src/WebJobs.Script.WebHost/App_Data/secrets/HttpTrigger.json

```diff
@@ -1,3 +0,0 @@
-﻿{
```

Comment:
Ok, I'll leave those. Though normally no one should need to be concerned with them since they will get generated